.github/workflows/build.yml

```
@@ -1,76 +1,117 @@
             
-      - name: Wait for healthy containers
+      - name: ⏳ Wait for healthy containers
         run: Tools/wait-for-healthy.sh
 
-      - name: Check database
+      - name: 🔄 Run database migrations
+        id: db_migration
+        env:
+          MYSQL_PWD: test
+        run: |
+          chmod +x Tools/db-migration.sh 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)